### PR TITLE
command line env hook

### PR DESCRIPTION
**Pull request by jihoonl**
zsh and tcsh do not work yet. bash works.

### CODE REVIEW

File: rocon_app_utilities/shells/env.tcsh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# No idea should be implemented later
```
Comment (bit-pirate) [inline rocon_app_utilities/shells/env.tcsh:1]: @jihoonl Did you create issues to track these missing implementations?

File: rocon_app_utilities/shells/env.zsh [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# No idea should be upgraded later
```
Comment (bit-pirate) [inline rocon_app_utilities/shells/env.zsh:1]: Same as above.

**Comment by jihoonl** [comment]
https://github.com/robotics-in-concert/rocon_app_platform/issues/184

**Comment by jihoonl** [comment]
https://github.com/robotics-in-concert/rocon_app_platform/issues/175

**Comment by jihoonl** [comment]
- The follow-up issue to implement zsh and tcsh has been created. #184 
- It implements #175 
- `rocon_app rawinfo` script has been introduced.
- `rocon_app list` now does not check ancestor conflict which arose during sprint review

**Comment by jihoonl** [comment]
- `rocon_app list` now provides rapp's `compatibilitiy`
- bug fix in indexer.

**Pull request merged**